libs/shared/l10n/src/lib/branding.ftl

```diff
@@ -43,6 +43,7 @@
 -product-firefox-cloud = Firefox Cloud
 -product-mozilla-monitor = Mozilla Monitor
 -product-firefox-relay = Firefox Relay
+-product-firefox-sync = Firefox Sync
```

Comment:
My understanding was that "Firefox Sync" is not treated as a brand term anymore and "sync" should just be treated as a verb. See: https://acorn.firefox.com/latest/content/word-list/s-t-D8DB7Wjd#section-sync-16